packages/calling-component-bindings/src/errorBarSelector.ts

```diff
@@ -43,8 +43,35 @@ export const errorBarSelector: ErrorBarSelector = createSelector(
     const activeErrorMessages: ActiveErrorMessage[] = [];
 
     // Errors reported via diagnostics are more reliable than from API method failures, so process those first.
-    if (diagnostics?.network.latest.networkReconnect?.value === 3) {
-      activeErrorMessages.push({ type: 'callingNetworkFailure' });
+    if (
+      diagnostics?.network.latest.networkReceiveQuality?.value === 3 ||
```

Comment:
What is the meaning of 2 and 3, is it a rating from 0-5?

Reply:
nit: update this to use the enum (i.e. NewtworkQuality.Poor instead of 2)

Reply:
Mad the same mistake in two PRs... :(